package-system/libpng/build_config.json

```diff
@@ -0,0 +1,67 @@
+{
+    "git_url": "https://git.code.sf.net/p/libpng/code",
+    "git_tag": "v1.6.37",
+    "package_name": "libpng",
+    "package_version": "1.6.37",
+    "package_url": "http://www.libpng.org/",
+    "package_license": "libpng-2.0",
+    "package_license_file": "LICENSE",
+    "cmake_find_source": "Findlibpng.cmake",
+    "cmake_find_target": "Findlibpng.cmake",
+    "Platforms": {
+        "Windows": {
+            "Windows": {
+                "cmake_generate_args": [
+                    "-DZLIB_ROOT=../../zlib-windows/zlib",
+                    "-DCMAKE_POLICY_DEFAULT_CMP0074=NEW",
+                    "-DPNG_SHARED=OFF",
+                    "-DPNG_TESTS=OFF",
+                    "-DPNG_DEBUG=OFF"
+                ]
+            },
+            "Android": {
+                "cmake_generate_args": [
+                    "-DZLIB_ROOT=../../zlib-android/zlib",
+                    "-DCMAKE_POLICY_DEFAULT_CMP0074=NEW",
+                    "-DPNG_SHARED=OFF",
+                    "-DPNG_TESTS=OFF",
+                    "-DPNG_DEBUG=OFF"
+                ]
+            }
+        },
+        "Darwin": {
+            "Mac": {
+                "cmake_generate_args": [
+                    "-DZLIB_ROOT=../../zlib-mac/zlib",
+                    "-DCMAKE_POLICY_DEFAULT_CMP0074=NEW",
+                    "-DPNG_SHARED=OFF",
+                    "-DPNG_TESTS=OFF",
+                    "-DPNG_DEBUG=OFF"
+                ]
+            },
+            "iOS": {
+                "cmake_generate_args": [
+                    "-DZLIB_ROOT=../../zlib-ios/zlib",
+                    "-DCMAKE_POLICY_DEFAULT_CMP0074=NEW",
+                    "-DPNG_SHARED=OFF",
+                    "-DPNG_TESTS=OFF",
+                    "-DPNG_DEBUG=OFF",
```

Comment:
I'm surprised this works withotu also specifying -DCMAKE_MACOSX_BUNDLE=OFF 
But if it does, great!